calico-cloud/network-policy/recommendations/learn-about-policy-recommendations.mdx

```diff
@@ -0,0 +1,145 @@
+---
+description: Learn more about policy recommendations.
+---
+
+# Learn more about policy recommendations
+
+## Big picture
+
+To use the [policy recommendation feature](policy-recommendations.mdx), you must have traffic between endpoints. In this tutorial we show you how recommendations are generated using flow logs in your cluster, for traffic to/from namespaces, network sets, private network IPs and public domains.
+
+### Configure felix for fast flow logs collection
+
+```bash
+kubectl patch felixconfiguration.p default -p '{"spec":{"flowLogsFlushInterval":"10s"}}'
+```
+
+### Apply the tutorial YAML, creating the necessary resources
+
+```bash
+kubectl create -f {{tutorialFilesURL}}/00-namespace.yaml
+```
+
+### Enable Policy Recommendation
+
+1. In the Manager UI left navbar, click on the **Policies** icon
+1. Select **Recommendations**
+1. Click on **Enable Policy Recommendations**
+
+Wait for the recommendations to be generated. 
+
+**Notes**:
+Unless otherwise configured, recommendations will take at least 2m30s to be generated, time equal to the default [Processing Interval](../../reference/resources/policyrecommendations.mdx#spec).
```

Comment:
```suggestion
Unless otherwise configured, recommendations will take at least 2m30s to be generated, which is the default time for [Processing Interval](../../reference/resources/policyrecommendations.mdx#spec).
```